src/nvim/screen.c

```diff
@@ -7070,7 +7070,7 @@ void draw_tabline(void)
   char_u      *p;
   int room;
   int use_sep_chars = (t_colors < 8
-                       );
+  );
```

Comment:
clearly, it universally hates lines beginning with end paren (but it doesn't dare to move the paren to the previous line)

Reply:
Same scenario as above. The code is "clearly" meant to be in one line, but isn't. I'm looking through the options for a solution. Worst case: we fix it manually once and be done with it.

Reply:
fixed